tasks/serve.js

```diff
@@ -22,7 +22,11 @@ module.exports = function(grunt) {
   // If no path is configured for Drush, fallback to the system path.
   var cmd = {cmd: Drupal.drushPath()};
 
-  var profile = grunt.config('config.serve.profile') || 'standard';
+  // `config.serve.profile` is deprecated.
+  if (grunt.config('config.serve.profile')) {
+    grunt.log.warn('The `serve.profile` parameter is deprecated and will be removed in future versions. Use `install.profile` instead.');
```

Comment:
Error message specified `install.profile`, config documentation uses `project.profile`. `project.profile` is better.